test/langtools/tools/javac/lambda/T8129740/Universe.java.out

```diff
@@ -69,29 +69,27 @@ class Universe {
                 }
                 
                 Planet(String name, int moonsCount) {
-                    this(name, moonsCount, java.lang.invoke.LambdaMetafactory.metafactory(name, Universe.Galaxy.this, Universe.Galaxy.SolarSystem.this));
+                    this(name, moonsCount, ()->{
```

Comment:
This test depends on the output of `printSource`, and since that output is what is left in the AST after lower, now it shows lambda expressions as well.